### PR TITLE
documentation publish failing

### PR DESCRIPTION
**Pull request by shubhamparikh927**
## Overview
- Updated the release.yml to remove the unwanted CP command for angular routing

## How it was tested
-  Tested the documentation publish job by temporarily updating the workflow to run on PR and commenting out npm publish job

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [ ] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [ ] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

**Review by semicolin — CHANGES_REQUESTED** [review]
I don't think this is the right solution. If you run a build locally and look at the `dist` dir, the application is now under a `/browser` subdir. We will need to append `/browser` to the paths in build-docs job.

I think the `cp` command is still needed. The Angular 17 documentation for [deploying to GitHub pages](https://v17.angular.io/guide/deployment#deploy-to-github-pages) still has this step:
> 5. When the build is complete, make a copy of docs/index.html and name it docs/404.html.

**Comment by shubhamparikh927** [comment]
> I don't think this is the right solution. If you run a build locally and look at the `dist` dir, the application is now under a `/browser` subdir. We will need to append `/browser` to the paths in build-docs job.
> 
> I think the `cp` command is still needed. The Angular 17 documentation for [deploying to GitHub pages](https://v17.angular.io/guide/deployment#deploy-to-github-pages) still has this step:
> 
> > 5. When the build is complete, make a copy of docs/index.html and name it docs/404.html.

**Comment by shubhamparikh927** [comment]
> I don't think this is the right solution. If you run a build locally and look at the `dist` dir, the application is now under a `/browser` subdir. We will need to append `/browser` to the paths in build-docs job.
> 
> I think the `cp` command is still needed. The Angular 17 documentation for [deploying to GitHub pages](https://v17.angular.io/guide/deployment#deploy-to-github-pages) still has this step:
> 
> > 5. When the build is complete, make a copy of docs/index.html and name it docs/404.html.

I have tested it as you suggested, It's working. So how should I have to publish the docuem

**Comment by shubhamparikh927** [comment]
> I don't think this is the right solution. If you run a build locally and look at the `dist` dir, the application is now under a `/browser` subdir. We will need to append `/browser` to the paths in build-docs job.
> 
> I think the `cp` command is still needed. The Angular 17 documentation for [deploying to GitHub pages](https://v17.angular.io/guide/deployment#deploy-to-github-pages) still has this step:
> 
> > 5. When the build is complete, make a copy of docs/index.html and name it docs/404.html.

I have tested it as you suggested, It's working, But now how should I have to publish the documentation as it runs on the new release. Should I have to release a new version for that?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
           cache: "npm"
       - run: npm ci
       - run: npm run build documentation -- --base-href=/charts-on-fhir/
-      - run: cp dist/apps/documentation/index.html dist/apps/documentation/404.html # Needed for Angular routing
+      - run: cp dist/apps/documentation/browser/index.html dist/apps/documentation/404.html # Needed for Angular routing
```
Comment (semicolin) [inline .github/workflows/release.yml:38]: Documentation site is not working: https://elimuinformatics.github.io/charts-on-fhir/
You also need to add /browser to the cp destination path and the upload-pages-artifact path

**Review by semicolin — CHANGES_REQUESTED** [review]

**Comment by semicolin** [comment]
> > I don't think this is the right solution. If you run a build locally and look at the `dist` dir, the application is now under a `/browser` subdir. We will need to append `/browser` to the paths in build-docs job.
> > I think the `cp` command is still needed. The Angular 17 documentation for [deploying to GitHub pages](https://v17.angular.io/guide/deployment#deploy-to-github-pages) still has this step:
> > > 5. When the build is complete, make a copy of docs/index.html and name it docs/404.html.
> 
> I have tested it as you suggested, It's working, But now how should I have to publish the documentation as it runs on the new release. Should I have to release a new version for that?

No need to release a new version. Just deploy it from this branch like you did before and verify that the documentation site is working.

**Review by vdeshkar — APPROVED** [review]

**Review by semicolin — APPROVED** [review]

**Pull request merged**